src/include/ZividPython/DataModelWrapper.h

```diff
@@ -145,6 +145,40 @@ namespace ZividPython
             static constexpr const char *value{ TypeName<T>::value };
         };
 
+        template<typename T>
+        struct TypeName<T, std::enable_if_t<Zivid::DataModel::IsDataModelRoot<T>::value>>
+        {
+        private:
+            class Helper
+            {
+                static constexpr auto concatenate() {
+                    constexpr auto prefix{ "_zivid." };
+                    constexpr size_t prefixLength{ std::char_traits<char>::length(prefix) };
+                    constexpr size_t nameLength{ std::char_traits<char>::length(T::name) };
+
+                    std::array<char, prefixLength + nameLength + 1> result{ '\0' };
+
+                    // TODO: Switch to std::copy_n when we can switch to C++20 and drop support for GCC 9.
```

Comment:
nitpick: maybe mention that `std::copy` is not used because it is not `constexpr` before C++20?